src/main/java/org/recommendation/model/Movie.java

```diff
@@ -0,0 +1,55 @@
+package org.recommendation.model;
+
+import java.util.List;
+
+public class Movie {
+    private String id;
+    private String title;
+    private String year;
+    private List<Genre> genres;
+
+    public Movie(final String id, final String title, final String year, final List<Genre> genres) {
+        this.id = id;
+        this.title = title;
+        this.year = year;
+        this.genres = genres;
```

Comment:
we should take deep copy so that if we change the Genre, Movie should not be changes